src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -774,19 +772,23 @@ void hiopVectorRajaPar::startingAtCopyToStartingAt(
 
   if(num_elems == 0)
     return;
-  auto& rm = umpire::ResourceManager::getInstance();
-  rm.copy(dest.data_dev_ + start_idx_dest, this->data_dev_ + start_idx_in_src, num_elems*sizeof(double));
+
+  //rm.copy(dest.data_dev_ + start_idx_dest, this->data_dev_ + start_idx_in_src, num_elems*sizeof(double));
+  //TODO: fix pointer arithmetic on host
+  dest.exec_space_.copy(dest.data_dev_+start_idx_dest, data_dev_+start_idx_in_src, num_elems, exec_space_);
 }
 
-void hiopVectorRajaPar::startingAtCopyToStartingAt_w_pattern(index_type start_idx_in_src,
-                                                             hiopVector& destination,
-                                                             index_type start_idx_dest,
-                                                             const hiopVector& selec_dest,
-                                                             size_type num_elems/*=-1*/) const
+template<class MEM, class POL>
+void hiopVectorRaja<MEM,POL>::
+startingAtCopyToStartingAt_w_pattern(index_type start_idx_in_src,
+                                     hiopVector& destination,
+                                     index_type start_idx_dest,
+                                     const hiopVector& selec_dest,
+                                     size_type num_elems/*=-1*/) const
 {
 #if 0  
```

Comment:
It isn't a part of the PR, but I really dislike these dangling `#if 0` (or `#if 1`) pre-processor directives. We should either remove them or control things via a real variable that is CMake toggleable.

Reply:
that code is commented out and not removed in case it is going to be needed  later on.